src/uk/ac/open/audio/streaming/StreamPlayerUI.java

```diff
@@ -116,27 +120,31 @@
 	 * @param dark Dark colour
 	 * @param light Light colour
 	 * @param faint Faint colour
+	 * @param image to show on pause
 	 */
 	public StreamPlayerUI(String startText,String stopText,
-			String cancelUploadText,Color dark,Color light,Color faint)
+			String cancelUploadText,Color dark,Color light,Color faint,Image pauseimage)
 	{
 		super(new BorderLayout(0,MARGIN));
 		setOpaque(false);
 		this.startText=startText;
 		this.stopText=stopText;
 		this.cancelUploadText=cancelUploadText;
 		this.buttonState=ButtonState.START;
+		pauseImage = pauseimage;
```

Comment:
Er, does this work? This seems like the one case where you do need this.pauseImage